packages/graphql-transformer-core/src/util/sanity-check.ts

```diff
@@ -295,6 +296,29 @@ export function cantEditLSIKeySchema(diff: Diff, currentBuild: DiffableProject,
   }
 }
 
+export function cantRemoveLSILater(diff: Diff, currentBuild: DiffableProject, nextBuild: DiffableProject) {
+  if (diff.kind === 'D' && diff.path.length === 6 && diff.path[5] === 'LocalSecondaryIndexes') {
```

Comment:
nit I would negate this logic and early return so the rest of the function isn't nested